### PR TITLE
docs: Changes the address prefix and example to major

### LINKED ISSUE #160: [PE-D][Tester E] Behaviour differ from UG for example command
This line of the UG: addstu nn/e0952224 n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 : Adds a student named John Doe to the contact book

in the UG:

 
The example commands gives an error:

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: qinxutan/ped#1

### LINKED ISSUE #165: [PE-D][Tester A] UG addstu description do not match application's
The addstu description in help has different compulsory fields. It also does not have the address field.

Because of this, I was mislead into thinking the other fields are failing in addstu, when in reality its because of this.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: LWS49/ped#7

### LINKED ISSUE #168: [PE-D][Tester E] UG command example given unable to execute
Below is the screenshot:

However once I removed address, the student can be added:

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: qinxutan/ped#3

### LINKED ISSUE #171: [PE-D][Tester E] UG Command example unable to execute
Similarly, for add student command:

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: qinxutan/ped#5

### LINKED ISSUE #184: [PE-D][Tester E] Unable to execute Add Student Command despite correct format
Correct format but add student command doesnt work.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: qinxutan/ped#9

### LINKED ISSUE #213: [PE-D][Tester D] User Guide Bug: no instruction on how to add or edit a major
Take a look at the screenshot below. There is no user guide on how to add a Major or edit a Major. 

This looks like a incomplete part in user guide under add and edit command.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: guohuang88/ped#11

### LINKED ISSUE #217: [PE-D][Tester A] Quick start Example error
As the address field has already been removed, the addstu example given does not work.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: LWS49/ped#19

### LINKED ISSUE #218: [PE-D][Tester A] Inconsistent graphic for Find
The message in UG uses the term "persons" while the message in application says "students".

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: LWS49/ped#20

### PR DESCRIPTION
**Pull request by Anant1902**
Resolves #160 
Resolves #168 
Resolves #171 
Resolves #184
Resolves #213
Resolves #217
Resolves #218
^ which all refer to outdated addstu command usage and example.
Partially resolves #165

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The latest version is compatible with older versions.
 
 A GUI similar to the below image should appear in a few seconds. Note how the app contains some sample data.
 
-<img src="images/main_window_ui.png" alt="Main Window UI on first launch" class="rounded-image"/>
+<img width="1470" alt="Screenshot 2024-04-06 at 10 30 53 PM" src="https://github.com/AY2324S2-CS2103T-F13-1/tp/assets/60879977/7752d686-20ff-4112-bb3c-a3441c22577d">
```
Comment (tituschewxj) [inline docs/UserGuide.md:46]: Consider putting the file inside the `docs/images` folder instead. And also name the file more appropriately. We should also delete the `images/main_window_ui.png` as it is outdated.

The image with is too large, and the image is cut off. The window size is not the same size as a new user's window.

To fix this, delete the `preferences.json` file and `data` directory and take another screenshot.

Additionally, we can include `class="rounded-image"` to round the corners of the image. It makes the screenshot look more polished.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ Format: `help`
 A new window is opened, so that it does not clutter up your main window. A GUI similar to the below image should appear.
 
 [//]: # (TODO: update this image once the help window is finalized in v1.4)
-<img src="images/help_window_ui.png" alt="Help Window UI" class="rounded-image" width="800px"/>
+<img width="857" alt="Screenshot 2024-04-06 at 10 32 22 PM" src="https://github.com/AY2324S2-CS2103T-F13-1/tp/assets/60879977/7da9f175-14df-422f-8f3c-34c3be7fca00">
```
Comment (tituschewxj) [inline docs/UserGuide.md:146]: This image is fine, but we should rename it, and move it into the project directories and delete the previous `help_window_ui.png`, and add rounded corners to this image.

Additionally, the image is cut off in the pdf version. I think the max width of an image to show correctly in the pdf would be 800, if the image is in a box. So we should reduce the width of this image to 800.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -143,7 +143,7 @@ Format: `help`
 A new window is opened, so that it does not clutter up your main window. A GUI similar to the below image should appear.
 
 [//]: # (TODO: update this image once the help window is finalized in v1.4)
```
Comment (tituschewxj) [inline docs/UserGuide.md:145]: Delete the todo since the image is updated.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -316,7 +316,8 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 <span class="semi-bold">#g#Example:##</span> `find alex david` returns `Alex Yeoh`, `David Li`, as seen in the image below.
 
 [//]: # (TODO: update this image)
-<img src="images/findAlexDavidResult.png" alt="result for 'find alex david" width="500px" class="rounded-image"/>
+<img width="618" alt="Screenshot 2024-04-06 at 10 33 37 PM" src="https://github.com/AY2324S2-CS2103T-F13-1/tp/assets/60879977/ebf1d631-b86a-41a6-b027-b27ce8fa049c">
```
Comment (tituschewxj) [inline docs/UserGuide.md:319]: Ditto. We can round the corners here too.

**Review by tituschewxj — CHANGES_REQUESTED** [review]
Thanks for this. I have left some nits regarding the screenshots. We can refer to #231 for tips on formatting the docs.

**Comment by tituschewxj** [comment]
Resolves #217 also

**Comment by Anant1902** [comment]
Also resolves #218

**Comment by Anant1902** [comment]
Resolves #213 too

**Comment by ForAeons** [comment]
Can just help to make sure all the parts edited by https://github.com/AY2324S2-CS2103T-F13-1/tp/pull/236/files is covered here too

**Comment by Anant1902** [comment]
> Can just help to make sure all the parts edited by https://github.com/AY2324S2-CS2103T-F13-1/tp/pull/236/files is covered here too

Hi yes, just updated for edit too! Aplogies for missing it out. Should cover everything now

**Comment by ForAeons** [comment]
> > Can just help to make sure all the parts edited by https://github.com/AY2324S2-CS2103T-F13-1/tp/pull/236/files is covered here too
> 
> Hi yes, just updated for edit too! Aplogies for missing it out. Should cover everything now

Line 199

**Comment by Anant1902** [comment]
> > > Can just help to make sure all the parts edited by https://github.com/AY2324S2-CS2103T-F13-1/tp/pull/236/files is covered here too
> > 
> > 
> > Hi yes, just updated for edit too! Aplogies for missing it out. Should cover everything now
> 
> Line 199

bro my command f is not working

**Comment by Anant1902** [comment]
Ok done @ForAeons can help check if g now?

**Review by tituschewxj — APPROVED** [review]
LGTM!

**Comment by Anant1902** [comment]
> LGTM!
> 
> Note
> 
> Putting `resolves` in a comment does not automatically resolve the issue when this is merged. Have to manually link it or put it in the main comment.
> 
> Additionally, each issue must have `resolves` before it: For example: `Resolves #160, resolves #168, resolves #171, resolves #184` will resolve all 4 issues but `Resolves #160, #168, #171, #184` will only resolve `#160` automatically.

Got it! Thanks a lot Titus.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -208,7 +207,7 @@ Format: `addstu n/NAME p/PHONE_NUMBER e/EMAIL nn/NUSNET_ID a/ADDRESS [t/TAG]…
 </box>
 
 Examples:
-* `addstu n/John Doe nn/e1234567 [p/98765432] [e/johnd@example.com] [a/John street, block 123, #01-01]`
+* `addstu n/John Doe nn/e1234567 p/98765432 e/johnd@example.com m/Computer Science`
 * `addstu n/Betsy Crowe nn/e01234567 t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
```
Comment (tituschewxj) [inline docs/UserGuide.md:211]: I noticed that this is wrong:
```suggestion
* `addstu n/Betsy Crowe nn/e01234567 t/friend e/betsycrowe@example.com m/Mathematics p/1234567 t/club`
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -196,7 +195,7 @@ Course code should follow the format "XX1234Y", Y is optional.
 ### Adding a student: `addstu`
 Adds a student to the contact book.
 
-Format: `addstu n/NAME p/PHONE_NUMBER e/EMAIL nn/NUSNET_ID a/ADDRESS [t/TAG]…​`
+Format: `addstu n/NAME p/PHONE_NUMBER e/EMAIL nn/NUSNET_ID m/MAJOR [t/TAG]…​`
```
Comment (tituschewxj) [inline docs/UserGuide.md:198]: From an unrelated issue, that it should follow the format inside our help window:
* I have added the issue here: #252 
```suggestion
Format: `addstu n/NAME nn/NUSNET [p/PHONE] [e/EMAIL] [m/MAJOR] [t/TAG]…​`
```

**Review by tituschewxj — COMMENTED** [review]
There's a few more things I spotted still while reviewing things again.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -517,16 +515,16 @@ If you move the application to a secondary screen, and later switch to using onl
...
-| **Help**           | `help`                                                                                                                                                                                            |
-| **Exit**           | `exit`                                                                                                                                                                                            |
+| Action             | Format, Examples                                                                                                                                                                       |
+|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add Student**    | `addstu n/NAME p/PHONE_NUMBER e/EMAIL nn/NUSNET_ID m/MAJOR [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com nn/e1234567 m/Computer Science t/friend t/colleague` |
```
Comment (ForAeons) [inline docs/UserGuide.md:520]: Phone and major fields are optional right

**Review by ForAeons — CHANGES_REQUESTED** [review]
Just one nit pick, the rest LGTM

**Review by ForAeons — APPROVED** [review]
LGTM

**Pull request merged**